src/main/java/com/springweb/dv_spring_web_mongo/controller/ProjectController.java

```diff
@@ -25,12 +27,12 @@ public ProjectDTO findProjectById(@PathVariable String id) {
     }
 
     @PostMapping()
-    public void addNewProject(@RequestBody ProjectDTO projectDTO) {
+    public void addNewProject(@Valid @RequestBody ProjectDTO projectDTO) {
```

Comment:
It would be nice if server returned error code with data what is the set of incorrect or missing fields